pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -95,6 +95,11 @@ type ScrapeConfigSpec struct {
 	// HonorLabels chooses the metric's labels on collisions with target labels.
 	// +optional
 	HonorLabels *bool `json:"honorLabels,omitempty"`
+	// HTTP scheme to use for scraping.
```

Comment:
(suggestion) let's reuse the description from upstream Prometheus.
```suggestion
	// Configures the protocol scheme used for requests.
```